src/main/java/hudson/tasks/Mailer.java

```diff
@@ -237,7 +237,11 @@ public static InternetAddress StringToAddress(String strAddress, String charset)
     public static DescriptorImpl DESCRIPTOR;
 
     public static DescriptorImpl descriptor() {
-        return Jenkins.get().getDescriptorByType(Mailer.DescriptorImpl.class);
+        final Jenkins jenkins = Jenkins.getInstanceOrNull();
+        if (jenkins == null) {
+            throw new IllegalStateException("Jenkins instance is not ready");
+        }
+        return jenkins.getDescriptorByType(Mailer.DescriptorImpl.class);
```

Comment:
```suggestion
        return Jenkins.get().getDescriptorByType(Mailer.DescriptorImpl.class);
```

Messed up the merge conflict resolution